crud/src/main/java/org/openmbee/mms/crud/controllers/elements/ElementsController.java

```diff
@@ -113,6 +126,54 @@ public ElementsResponse createOrUpdateElements(
         throw new BadRequestException(response.addMessage("Empty"));
     }
 
+    @PostMapping(value = "/stream", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
can comment this function out for now so it doesn't show up in generated swagger